BlackNectar/BlackNectar/AppDelegate.swift

```diff
@@ -7,16 +7,31 @@
 //
 
 import UIKit
+import AromaSwiftClient
```

Comment:
It is a good practice to order imports Alphabetically whenever possible. 

Some method to the madness! 🙊